apps/frontend/src/pages/quiz/libs/components/analyzing/libs/helpers/create-animated-spans/create-animated-spans.helper.ts

```diff
@@ -0,0 +1,27 @@
+import React from "react";
```

Comment:
will React.createElement work without this import?

Reply:
No, because its not a tsx/jsx file.

Reply:
`"error 'React' refers to a UMD global, but the current file is a module. Consider adding an import instead."`